api/api_post.py

```diff
@@ -11,3 +13,10 @@
 
 
 # Routes go here
+@api_post.route('/api/user_import', methods=['POST'])
+def user_import():
+    name=request.args.get('name')
+    password=request.args.get('password')
+    user = User(name, password)
+    user.input_user()
+    return user.name
```

Comment:
This needs to be surrounded by try-except - It will break if someone forgot to add `name` or `password` parameters to request.
Also, please add log that we are accessing this route, to back-trace in logs